src/cljs/harja/tiedot/urakka/urakka.cljs

```diff
@@ -61,6 +61,12 @@
     (number? (-> arvo js/parseFloat))
     arvo))
 
+(defn negatiivinen-numero [arvo]
+  (when
+    (and (number? (-> arvo js/parseFloat))
+      (not (pos? arvo)))
```

Comment:
neg? on myös

Reply:
(neg? x) != (not (pos? x), halusin sallia myös nollan validoinnissa.

Reply:
aivan totta 👍